libraries/core_libs/storage/src/storage.cpp

```diff
@@ -674,6 +678,27 @@ void DbStorage::addPbft2TPlus1ToBatch(uint64_t pbft_round, size_t pbft_2t_plus_1
   insert(write_batch, DbStorage::Columns::pbft_round_2t_plus_1, toSlice(pbft_round), toSlice(pbft_2t_plus_1));
 }
 
+size_t DbStorage::getPbftSortitionThreshold(uint64_t period) {
+  auto status = lookup(toSlice(period), Columns::period_pbft_sortition_threshold);
+
+  if (!status.empty()) {
+    size_t value;
+    memcpy(&value, status.data(), sizeof(size_t));
+    return value;
+  }
+
+  return 0;
+}
+
+void DbStorage::savePbftSortitionThreshold(uint64_t period, size_t sortition_threshold) {
+  insert(Columns::period_pbft_sortition_threshold, toSlice(period), toSlice(sortition_threshold));
+}
+
+void DbStorage::addPbftSortitionThresholdToBatch(uint64_t period, size_t sortition_threshold, Batch& write_batch) {
+  insert(write_batch, DbStorage::Columns::period_pbft_sortition_threshold, toSlice(period),
+         toSlice(sortition_threshold));
+}
+
```

Comment:
we do not need this at all

Reply:
Since PBFT finalized blocks are async with execution of final chain. We need this

Reply:
Removed in another PR https://github.com/Taraxa-project/taraxa-node/pull/1710
Let me test it

Reply:
Removed